patchwork/steps/AnalyzeImpact/AnalyzeImpact.py

```diff
@@ -84,6 +84,9 @@ def __init__(self, inputs: dict):
         self.inputs = inputs
 
     def run(self) -> dict:
+        if self.inputs.get("debug") is not None:
```

Comment:
Instead of every implemented step having to check for this input key perhaps this should be managed in the abstract class `Step` at step.py instead.

Reply:
Please do not let it be for now. Thank you.